examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -99,6 +99,67 @@ databases: []
 #      "*": "*"
 databaseResources: []
 
+################################################################
+# Values that must be provided if Windows Desktop access is enabled.
+################################################################
+
+# AD or non-AD configuration
+
+# (optional) Determines whether desktop sessions will show a user-selected wallpaper
+# vs a system-default, single-color wallpaper. For improved visual performance,
+# set this to false (its default value).
+#showWindowsDesktopWallpaper: false
+
+# (optional) ldap contains configuration keys used when connecting Teleport
+# to an Active Directory domain. This enables the discovery service for
```

Comment:
This is not true. LDAP discovery is optional. The LDAP config is required (for AD)

Reply:
Updated